oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-service.json

```diff
@@ -17,317 +17,345 @@
+                              "title":"Connects"
+                           }
+                        },
+                        {
+                           "x":18,
+                           "y":9,
+                           "w":6,
+                           "h":10,
+                           "i":"10",
+                           "type":"Widget",
+                           "metricTypes":[
+                              "readMetricsValues"
+                           ],
+                           "metrics":[
+                              "meter_mysql_slow_queries_rate"
+                           ],
+                           "graph":{
+                              "type":"Line",
+                              "step":false,
+                              "smooth":false,
+                              "showSymbol":false,
+                              "showXAxis":true,
+                              "showYAxis":true
+                           },
+                           "widget":{
+                              "title":"Slow Queries Trend (queries per second)"
+                           }
+                        },
+                        {
+                           "x":12,
+                           "y":19,
+                           "w":6,
+                           "h":10,
+                           "i":"11",
+                           "type":"Widget",
+                           "metricTypes":[
+                              "readMetricsValues",
+                              "readMetricsValues"
+                           ],
+                           "metrics":[
+                              "meter_mysql_connection_errors_internal",
+                              "meter_mysql_connection_errors_max_connections"
+                           ],
+                           "graph":{
+                              "type":"Line",
+                              "step":false,
+                              "smooth":false,
+                              "showSymbol":false,
+                              "showXAxis":true,
+                              "showYAxis":true
+                           },
+                           "widget":{
+                              "title":"Connection Errors"
+                           }
+                        }
+                     ]
+                  },
                   {
-                     "calculation":"byteToMB"
+                     "name":"Slow Statements",
+                     "children":[
+                        {
+                           "x":0,
+                           "y":0,
+                           "w":24,
+                           "h":29,
+                           "i":"0",
+                           "type":"Widget",
+                           "metricTypes":[
+                              "readSampledRecords"
+                           ],
+                           "metrics":[
+                              "top_n_database_statement"
```

Comment:
Do we support to set N for sampled record on UI?
As you use a whole page to show. Top 50 seems better.

Reply:
> Do we support to set N for sampled record on UI? As you use a whole page to show. Top 50 seems better.

We support that, and default is 10.
> As you use a whole page to show. Top 50 seems better.
I get it.